schedule/yast/lvm_raid1/lvm+raid1_opensuse.yaml

```diff
@@ -32,3 +30,8 @@ schedule:
   - installation/grub_test
   - installation/first_boot
   - console/validate_lvm_raid1
+test_data:
+  system_role:
+    default: none
+    selection: 'Des&ktop with KDE Plasma'
```

Comment:
Hmm, we have talked to @OleksandrOrlov about it, we should act on server side to make it shortcut independent. As of now we could do that on client side, as I'm not sure we will be able to get it in SLE 15 SP3. In the worst case we will have to maintain 2 versions of `test_data`, so we should evaluate risks before applying it.
See https://progress.opensuse.org/issues/90098

Reply:
good to know, that makes sense to fix it in the server. I could not fix it in the client, it is easy to remove '&' when reading but for the POST we need to know where the symbol is, so for simplicity of the test data I used the same for both, even for default we could have filtered it.

Reply:
On the client side we can similar solution as in ruby client, that we search by regexp and then use found value for the POST requests. Anyway, we should keep it in mind, as we will have same issues as before if shortcuts do change.